### PR TITLE
Pl minor docupdates

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/20_References/entity-deletion-reference.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ In some cases, you might try to delete an entity, but the entity is used by anot
 
 Harness will identify that you do not have permission to edit the other entity, and so you cannot delete this entity.
 
-### Deletion of Roles
+### Deletion of roles
```
Comment (SmritiSatya) [inline docs/platform/20_References/entity-deletion-reference.md:48]: Not sure if what I am thinking is right, but can we rephrase this to 'Role deletion' or 'Role deletions'?

File: docs/platform/20_References/entity-deletion-reference.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Role Id has the following deletion rules:
 * If the deleted role effects active users, they are prevented from performing the operations granted by the removed role’s permissions.   
 For example, if a user was assigned to a role that had the Create/Edit Pipeline permission, and the role is removed, then the user can no longer create or edit Pipelines.
 
-### Deletion of Users
+### Deletion of users
```
Comment (SmritiSatya) [inline docs/platform/20_References/entity-deletion-reference.md:59]: Same- 'User deletion' or 'User deletions'?

File: docs/platform/20_References/entity-deletion-reference.md (hunk 1 above)
Comment (rashminandasahoo) [reply docs/platform/20_References/entity-deletion-reference.md]: Done.

**Review by SmritiSatya — APPROVED** [review]
